### ISSUE TITLE
Unable to build from source - unresolvable dependency

### ISSUE THREAD

**Issue by scpketer** [BanManagement/BanManager#797]
# Issue report
I'm trying to build the plugin from source, however, Maven fails to build it due to unresolvable dependency of `net.gravitydevelopment.updater`:

```
[ERROR] Failed to execute goal on project BanManager: Could not resolve dependencies for project me.confuser:BanManager:jar:6.0.2: Failed to collect dependencies at net.gravitydevelopment.updater:updater:jar:3.0.3: Failed to read artifact descriptor for net.gravitydevelopment.updater:updater:jar:3.0.3: Could not transfer artifact net.gravitydevelopment.updater:updater:pom:3.0.3 from/to dustplanet-releases (https://repo.dustplanet.de/artifactory/libs-release-local): Failed to transfer file https://repo.dustplanet.de/artifactory/libs-release-local/net/gravitydevelopment/updater/updater/3.0.3/updater-3.0.3.pom with status code 503
```

The same problem persists [at your CI](https://jitpack.io/com/github/BanManagement/BanManager/v6.0.2/build.log), and thus I cannot use this plugin as a dependency (using JitPack), either.

Possible solution: include Updater jar right into plugin jar?

**Comment by confuser** [comment]
Is there a reason you need to use JitPack? A prebuilt jar is available via maven https://ci.frostcast.net/plugin/repository/everything/

**Comment by scpketer** [comment]
Didn't know about that. But still - sources fail to build.

**Comment by confuser** [comment]
Resolved in #795
